prereise/gather/winddata/hrrr/downloader.py

```diff
@@ -0,0 +1,18 @@
+import shutil
+
+import requests
+
+
+class Downloader:
+    """Class that holds downloading functionality"""
+
+    @staticmethod
+    def download(url, file, headers={}):
```

Comment:
In general, I would avoid [mutable default arguments](https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument). The alternative would be to default to `None`, followed by

```
headers = {} if headers is None else headers
```

Reply:
TIL, will do